lib/initializer.rb

```diff
@@ -0,0 +1,47 @@
+require_relative './game.rb'
+require_relative './board/board.rb'
+require_relative './board/board_checker.rb'
+require_relative './players/ai.rb'
+require_relative './players/human.rb'
+require_relative './players/player.rb'
+require_relative './ui/user_interface.rb'
+require_relative './ui/script.rb'
+
+def initializer(size, empty_char)
+  board = Board.new(size, empty_char)
+  checker = BoardChecker.new(size, empty_char)
+  script = Script.new
+  user_interface = UserInterface.new(board, script)
+
+  user_interface.welcome
+  mode_selection = user_interface.what_mode?
+
+  unless(mode_selection == 'computerVScomputer')
```

Comment:
have you considered migrating this string somewhere else?
(http://deviq.com/magic-strings/)